docs/modules/reference/pages/glossary.adoc

```diff
@@ -28,35 +36,53 @@ There, users can find knowledge base articles, support, announcements, and gener
 Discovery:: A process that uses ICMP to find network devices and services in your network and then automatically adds them to monitoring.
 (See xref:operation:provisioning/introduction.adoc#discovery-auto[Provisioning].)
 
-Docker:: Docker is an open-source virtualization service consisting of three components: Docker Compose (a tool for defining and running multi-container Docker applications), Docker Swarm (a tool providing native clustering functionality for Docker containers), and Docker Volume (a tool that allows for the independent persistence of data). 
+Docker:: Docker is an open-source virtualization service consisting of three components: Docker Compose (a tool for defining and running multi-container Docker applications), Docker Swarm (a tool providing native clustering functionality for Docker containers), and Docker Volume (a tool that allows for the independent persistence of data).
 (See https://docs.docker.com/[Docker].)
 
 Dominion:: The service on an OpenNMS core that controls the operation of Minions.
-(See xref:reference:configuration/minion-confd/minion-confd.adoc#dominion [Configuring Minion via Confd].)
+(See xref:reference:configuration/minion-confd/minion-confd.adoc#dominion[Configuring Minion via Confd].)
 
-Elasticsearch:: Elasticsearch is a search engine based on the Lucene library. 
+Elasticsearch:: Elasticsearch is a NoSQL database (See https://azure.microsoft.com/en-us/overview/nosql-database/[What is NoSQL?]) with an integrated search engine based on the Lucene library.
 OpenNMS persists/forwards certain data to Elasticsearch to store flow records and warehouse events and alarm history for long-term storage of this data.
 (See xref:operation:elasticsearch/introduction.adoc#elasticsearch[Elasticsearch Integration].)
 
-Flows:: Summaries of network traffic sent by network devices. 
+Events:: Structured historical records of activities in {page-component-title} and the nodes, interfaces, and services it manages.
+(See xref:operation:events/anatomy-events.adoc[Anatomy of an Event].)
+
+Flows:: Summaries of network traffic sent by network devices.
 (See xref:operation:flows/introduction.adoc[Flows].)
 
 Foreign Source:: A data source hosted outside of your local network.
 To customize monitoring of nodes in a foreign source, you can set a foreign source definition with your desired detectors and policies.
-(See xref:operation:provisioning/getting-started.adoc#foreign-source-definition[Foreign source definition].)
+(See xref:operation:provisioning/getting-started.adoc#foreign-source-definition[Foreign Source Definition].)
+
+Geolocation:: The actual physical location of a node monitored by {page-component-title}.
+Nodes can be manually assigned a location, or the location can be automatically determined using the GeoIP Provisioning Adapter (See xref:reference:provisioning/adapters/geoip.adoc[GeoIP]).
```

Comment:
```suggestion
Nodes can be manually assigned a location, or the location can be automatically determined using the ref:reference:provisioning/adapters/geoip.adoc[GeoIP Provisioning Adapter].
```
Just tightening this up a bit.

Reply:
Grr. Cut and paste error. It should be xref.

Reply:
```suggestion
Nodes can be manually assigned a location, or the location can be automatically determined using the xref:reference:provisioning/adapters/geoip.adoc[GeoIP Provisioning Adapter].
```